### PR TITLE
[Website]:  Split custom.scss file into multiple files

### LINKED ISSUE #3820: [Website] Create global theme for styling components
### Describe your idea

Detox website's Figma: [https://www.figma.com/file/QGhhB7djq0OxvLzU…](https://www.figma.com/file/QGhhB7djq0OxvLzULhs7u5/Detox)

- [ ] Make a global styling PR that will fix the fonts, colors, etc. in 'custom.css'
- [ ] Current website we are mixing the old and new design and some components are using old fonts and some are using the new fonts (like in the admonitions)
- [ ] make the most global docusaurus theme related cusomizations in a PR
- [ ] Next step to then go back and style the specific components using old styles to match new Figma styles

### PR DESCRIPTION
**Pull request by moutro**
## Description

Subtask of #3820 

- This pull request addresses the issue described here: 

- **_Split_** custom.scss file into multiple .scss files into separate .scss files.
- Import all back into custom.scss file.

Note: Live Sass Watcher Extension added a comment at the bottom of all CSS files. Left them there for now but please let me know if you want me to remove. This is ths reason there looks to be many files modified in this PR.

cc: @asafkorem

### CODE REVIEW

**Review by noomorph — CHANGES_REQUESTED** [review]
There is a problem. I expected that all the `*.css` would have gone away. Instead, I see good old CSS files along with the new SCSS files.

@moutro, could you explain how you ended up with these quasi-duplicates?

File: website/src/css/_benefits-section.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+[data-theme='dark'] .benefits-header {
+  color: #ffffff;
+}
+
+.social-buttons {
+  justify-content: flex-end;
+  display: flex;
+}
+
+.flex-jc-fe {
```
Comment (noomorph) [inline website/src/css/_benefits-section.scss:21]: Does it really belong to the benefits section? Looks generic on a first glance.

File: website/src/css/custom.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@import 'benefits-section';
+@import 'footer-socials';
+@import 'footer';
+@import 'header-github';
+@import 'navbar';
+@import 'showcase-link';
+@import 'showcase-section';
+@import 'utilities';
+
+.docusaurus-highlight-code-line {
```
Comment (noomorph) [inline website/src/css/custom.scss:14]: This can be extracted to `_markdown.scss`.

**Comment by moutro** [comment]
> There is a problem. I expected that all the `*.css` would have gone away. Instead, I see good old CSS files along with the new SCSS files.
> 
> @moutro, could you explain how you ended up with these quasi-duplicates?

@noomorph - I believe that what I did was to keep the `*.css` files includig the `custom.css` file in the projects but should have deleted all of them since I am using the SASS plugin and also including it in the config file to point to the `custom.scss` file.  Based on this, I should have deleted the css files and will do so now. Thanks!

File: website/src/css/_benefits-section.scss (hunk 1 above)
Comment (moutro) [reply website/src/css/_benefits-section.scss:21]: @noomorph  - This is being using just in the Homepage SubHeader for the social icons. It's why I put it here. Is it not correct?
image

File: website/src/css/custom.scss (hunk 2 above)
Comment (moutro) [reply website/src/css/custom.scss:14]: @noomorph - do you mean lines 14 - 18 and 24 - 26?

Comment (noomorph) [reply website/src/css/custom.scss:14]: Looks like it.

As for`[data-theme='light'] .banner {`, I doubt it is used at all.

Comment (moutro) [reply website/src/css/custom.scss:14]: I think it belongs to the UA banner

Comment (noomorph) [reply website/src/css/custom.scss:14]: Maybe someone it belongs by intent, but not by any actual effect. See:

image

The class `banner_SLir` != `banner`, so this rule looks useless.

Comment (moutro) [reply website/src/css/custom.scss:14]: Thanks. I will remove it since it's not doing anything.

**Review by noomorph — APPROVED** [review]

**Pull request merged**